src/fauna/fql/demo.fql

```diff
@@ -36,10 +36,9 @@ Credentials.byDocument(researcher).login("changeme")
 
 Record.all()
 
-// researcher accesses a record 
-
-AccessRecord(Record("402219589133402180"), Query.identity(), "Disclosure" )
+// researcher accesses Jean King's records
 
+PatientRecords(Patient("402223095656482304"), "Disclosure")
```

Comment:
Note that you might need to change the patient id during the demo. Make sure to copy it when you read the patient first time...

Reply:
roger